static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -1213,13 +1259,31 @@ function UploadedSourceMapWithCorrectDebugIdChecklistItem({
       <CheckListInstruction type="muted">
         <h6>{t('No Artifacts Uploaded')}</h6>
         <p>
-          {tct(
-            "You didn't upload any artifacts with debug IDs yet. Read the [link:Sentry Source Maps Documentation] to learn how to inject Debug IDs into your build artifacts and how to upload them to Sentry.",
-            {
-              link: <ExternalLinkWithIcon href={sourceMapsDocLinks.sourcemaps} />,
-            }
-          )}
+          {tct('You did not upload any artifacts with debug IDs yet.', {
+            link: <ExternalLinkWithIcon href={sourceMapsDocLinks.sourcemaps} />,
+          })}
```

Comment:
same here. Also if we are going to repeat this multiple times, maybe it is worth creating a reusable component